crates/console/src/console_dbg.rs

```diff
@@ -6,16 +6,15 @@
 #[macro_export]
 macro_rules! console {
     () => {
-        $crate::log!(::std::format!("[{}:{}]", ::std::file!(), ::std::line!()));
+        $crate::log!(
+            ::std::format!("%c[{}:{}] ", ::std::file!(), ::std::line!()),
+            "font-weight: bold",
+        );
     };
     ($val:expr $(,)?) => {
-        // Use of `match` here is intentional because it affects the lifetimes
-        // of temporaries - https://stackoverflow.com/a/48732525/1063961
-        match $val {
-            tmp => {
-                $crate::log!(::std::format!("[{}:{}] {} =", ::std::file!(), ::std::line!(), ::std::stringify!($val)), &tmp);
-                tmp
-            }
```

Comment:
This is no longer necessary, existed for two reasons:
1. expressed called twice 
2. align lifetimes of multiple expressions 

1 won't happen anymore because now both macros only run the expression once and bind them to a variable.
2 is not necessary. It's only needed in cases like assert_eq.